modules/readme/Makefile

```diff
@@ -3,6 +3,8 @@ export README_FILE ?= README.md
 export README_YAML ?= README.yaml
 export README_TEMPLATE_FILE ?= $(BUILD_HARNESS_PATH)/templates/README.md
 export README_TEMPLATE_YAML ?= $(BUILD_HARNESS_PATH)/templates/README.yaml
+export USED_DATASOURCE ?= $(if $(datasource),$(datasource),$(shell pwd)/$(README_YAML))
```

Comment:
Where does is `$(datasource)` defined? I would expect to see `datasource =` somewhere.

Reply:
The user can just specify `make readme README_TEMPLATE_YAML=foobar.yaml` if they want to override, so unsure what `datasource` gives us.